test/test.cc

```diff
@@ -26,6 +26,59 @@
 #include <sys/stat.h>
 
 namespace {
+bool json_deep_equality(const rapidjson::Value& j1, const rapidjson::Value& j2) {
```

Comment:
this is a bit annoying but actually pretty useful. not sure if this is not implemented somewhere already in some test file :sweat_smile: